index.html

```diff
@@ -371,6 +371,23 @@ <h2 class="logo-scratchx">ScratchX</h2>
                         </section>
                     </li>
                     <li>
+                        <a href="https://ericrosenbaum.github.io/spotify-extension/" target="_blank">
```

Comment:
It looks like most of the other links go straight into the extension. Any thoughts on making this consistent?